src/components/listings/ImageGridItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import { ImageBackground, Pressable } from 'react-native'
+import Animated, {
+  useSharedValue,
+  withDelay,
+  withTiming,
+} from 'react-native-reanimated'
+import Icon from '../Icon'
+import { IconSize } from '@/types/icon'
+import { useColours } from '@/hooks/useTailwind'
+import { ImageGridItemProps } from '@/types/imageGrid'
+
+const DURATION = 200
+const DELAY = 50
+
+function ImageGridItem(props: ImageGridItemProps) {
```

Comment:
Can we set this up to accept a default "selected" state via props?

Reply:
👍